gate-web/config/gate.yml

```diff
@@ -27,14 +27,17 @@ services:
             shortDescription: v14.04
             detailedDescription: Ubuntu Trusty Tahr v14.04
             packageType: DEB
+            vmTypes: [ 'hvm', 'pv' ]
```

Comment:
I legit have no idea where these images came from or if they have any relation to reality. Pretty much just adding random lists as it seems like a nice way of self documenting.